packages/rps/src/redux/auto-opponent/__tests__/auto-opponent.test.ts

```diff
@@ -15,7 +15,8 @@ const reducer = combineReducers({
   openGames: openGamesReducer,
 });
 
-it(
+// TODO: Get this working with a mock for channel-client
+it.skip(
```

Comment:
Could this be a goal for @cf19drofxots with the `FakeChannelClient`?

Reply:
Yup, I think so!